go.mod

```diff
@@ -6,8 +6,8 @@ toolchain go1.22.1
 
 require (
 	github.com/joho/godotenv v1.5.1
+	github.com/supabase-community/auth-go v1.0.0
```

Comment:
v1.0.0 is using old path, can use the latest version.

Also, need to run `go mod tidy` to update the `go.sum`